app/models/comment.php

```diff
@@ -42,6 +63,72 @@ public function write()
         );
 
         $db = DB::conn();
-        $db->insert('comment',$params);
+        $db->insert(self::TABLE,$params);
```

Comment:
Same as my comment [here](https://github.com/renzosunico/MyClassroom/pull/2/files#r39120524)
Please check your codes and apply to all.